### PR TITLE
⚡Added CodeBlockParser

### PR DESCRIPTION
**Pull request by RajnishAnand**
⚡Updated CodeBlockParser regex
⚡Added Readme
⚡Add in test-bot Command
⚡Updated package-lock.json
⚡Added my bot token 😂 kidding

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -167,8 +169,31 @@ confirm(options);
 
 > _Note: You need some intents to enable these features as per discord.js@v13. For reference see [index.ts](./src/test-bot/index.ts)_
 
-# Contributing
```
Comment (KrishAgarwal2811) [inline README.md:170]: You removed this heading but never added it again :chadthink:

File: README.md [hunk 2]
```diff
@@ -167,8 +169,31 @@ confirm(options);
...
+A code Block Parser to extract language and code out of code block
 
+**Parameters:**
+- **string** - string containing codeBlock
+
+**return:**
+- `[Object]`
+  - `lang ?: string`
+  - `code ?: string`
+> Exemple :
```
Comment (KrishAgarwal2811) [inline README.md:183]: typo

**Review by KrishAgarwal2811 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (RajnishAnand) [reply README.md:170]: :chadthink: i don't remember tho 😅

Comment (KrishAgarwal2811) [reply README.md:170]: no it isn't there :chadthink:

**Review by KrishAgarwal2811 — CHANGES_REQUESTED** [review]

**Comment by RajnishAnand** [comment]
fixed now 😅 `# contribution` added
and typo : `Exemple` →`Example` phixed

**Pull request merged**